articles/location-based-services/toc.yml

```diff
@@ -46,7 +46,7 @@
   - name: Azure Roadmap
     href: https://azure.microsoft.com/roadmap/?category=internet-of-things
   - name: MSDN forum
-    href: https://social.msdn.microsoft.com/Forums/azure
+    href: https://social.msdn.microsoft.com/Forums/en-US/home?forum=azurelbs
```

Comment:
Can you update your PR to use the following URL (without the locale identifier):
`https://social.msdn.microsoft.com/Forums/home?forum=azurelbs`
Thanks!

Reply:
@dominicbetts , @kgremban, I think I found a way to remove locale info by adding azure in the URL, and it will be:

https://social.msdn.microsoft.com/Forums/azure/home?forum=azurelbs